gensim/models/wrappers/fasttext.py

```diff
@@ -298,8 +299,16 @@ def load_dict(self, file_handle, encoding='utf8'):
                 char_byte = file_handle.read(1)
             word = word_bytes.decode(encoding)
             count, _ = self.struct_unpack(file_handle, '@qb')
-            assert self.wv.vocab[word].index == i, 'mismatch between gensim word index and fastText word index'
-            self.wv.vocab[word].count = count
+
+            if word != "__label__":
```

Comment:
Please add a note briefly clarifying this, since it's a hack-ish fix and likely to confuse any future developers.